WordPress/Classes/Services/ReaderPostService.m

```diff
@@ -88,18 +93,38 @@ - (void)fetchPostsForTopic:(ReaderAbstractTopic *)topic
                                   }];
 }
 
+
+- (void)updateTopic:(NSManagedObjectID *)topicID withAlgorithm:(NSString *)algorithm
```

Comment:
Maybe rename `topicID` as `topicObjectID` for clarity? Although the type does infer.